packages/cli/src/util/link/add-git-connection.ts

```diff
@@ -96,12 +104,13 @@ async function addMultipleGitRemotes(
   org: Org,
   project: Project,
   remoteUrls: Dictionary<string>,
-  settings: ProjectSettings
+  settings: ProjectSettings,
+  autoConfirm: Boolean
 ) {
-  client.output.log('Found multiple Git remote URLs in Git config.');
-  const remoteUrlOrOptions = await promptGitConnectMultipleUrls(
-    client,
-    remoteUrls
-  );
+  let remoteUrlOrOptions = autoConfirm ? Object.values(remoteUrls)[0] : '';
+  if (!remoteUrlOrOptions) {
+    client.output.log('Found multiple Git remote URLs in Git config.');
+    remoteUrlOrOptions = await promptGitConnectMultipleUrls(client, remoteUrls);
+  }
   return handleOptions(remoteUrlOrOptions, client, org, project, settings);
```

Comment:
**suggestion (non-blocking):** rearrange logic for clarity

The use of `remoteUrlOrOptions` for multiple values is confusing.

```suggestion
  let remoteUrl: string | undefined;
  if (autoConfirm) {
    remoteUrl = Object.values(remoteUrls)[0];
  } else {
    client.output.log('Found multiple Git remote URLs in Git config.');
    remoteUrl = await promptGitConnectMultipleUrls(client, remoteUrls);
  }
  return handleOptions(remoteUrl, client, org, project, settings);
```